packages/portal/spec-renderer-mini/src/components/OperationsListItem.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <button
+    class="item"
+    :class="{ 'item--selected': isSelected, [`item--method-${item.method}`]: true }"
+    :data-testid="`mini-spec-item-${item.path}`"
+    type="button"
+    @click="emit('click', item)"
+  >
```

Comment:
question: any reason you're using a regular button here instead of `KButton`?

Reply:
The button here is used specifically for accessibility semantics. Using KButton would not have any added value, in my opinion. The same applies for other native `<button>` usages in this PR